services/web/pkg/config/options.go

```diff
@@ -0,0 +1,65 @@
+package config
+
+// Options are the option for the web
+type Options struct {
+	HomeFolder               string          `json:"homeFolder" yaml:"homeFolder" env:"WEB_OPTION_HOME_FOLDER" desc:"You can specify a folder that is used when the user navigates 'home'. Navigating home gets triggered by clicking on the 'All files' menu item. The user will not be jailed in that directory. It simply serves as a default location. You can either provide a static location, or you can use variables of the user object to come up with a user specific home path. This uses twig template variable style and allows you to pick a value or a substring of a value of the authenticated user. Examples are '/Shares', '/{{.Id}}' and '/{{substr 0 3 .Id}}/{{.Id}'."`
+	OpenAppsInTab            bool            `json:"openAppsInTab" yaml:"openAppsInTab" env:"WEB_OPTION_OPEN_APPS_IN_TAB" desc:"Configures whether apps and extensions generally should open in a new tab. Defaults to false."`
+	DisablePreviews          bool            `json:"disablePreviews" yaml:"disablePreviews" env:"WEB_OPTION_DISABLE_PREVIEWS" desc:"Set this option to 'true' to disable previews in all the different file listing views. The only list view that is not affected by this is the trash bin, as that doesn't allow showing previews at all."`
+	PreviewFileMimeTypes     []string        `json:"previewFileMimeTypes" yaml:"previewFileMimeTypes" env:"WEB_OPTION_PREVIEW_FILE_MIMETYPES" desc:"Specifies which mimeTypes will be previewed in the ui. For example to only preview jpg and text files set this option to ['image/jpeg', 'text/plain']."`
+	AccountEditLink          AccountEditLink `json:"accountEditLink" yaml:"accountEditLink"`
+	DisableFeedbackLink      bool            `json:"disableFeedbackLink" yaml:"disableFeedbackLink" env:"WEB_OPTION_DISABLE_FEEDBACK_LINK" desc:"Set this option to 'true' to disable the feedback link in the topbar. Keeping it enabled (value 'false' or absence of the option) allows ownCloud to get feedback from your user base through a dedicated survey website."`
+	FeedbackLink             FeedbackLink    `json:"feedbackLink" yaml:"feedbackLink"`
+	SharingRecipientsPerPage int             `json:"sharingRecipientsPerPage" yaml:"sharingRecipientsPerPage" env:"WEB_OPTION_SHARING_RECIPIENTS_PER_PAGE" desc:"Sets the amount of users shown as recipients in the dropdown when sharing resources. Default amount is 200."`
+	Sidebar                  Sidebar         `json:"sidebar" yaml:"sidebar"`
+	RunningOnEOS             bool            `json:"runningOnEos" yaml:"runningOnEos" env:"WEB_OPTION_RUNNING_ON_EOS" desc:"Set this option to 'true' if running on an [EOS storage backend](https://eos-web.web.cern.ch/eos-web/) to enable its specific features. Defaults to 'false'."`
```

Comment:
```suggestion
	RunningOnEOS             bool            `json:"runningOnEos" yaml:"runningOnEos" env:"WEB_OPTION_RUNNING_ON_EOS" desc:"Set this option to 'true' if running on an EOS storage backend (https://eos-web.web.cern.ch/eos-web/) to enable its specific features. Defaults to 'false'."`
```
We cant use markdown for links